services/QuillLMS/app/models/concerns/translation_prompts.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module TranslationPrompts
+  extend ActiveSupport::Concern
+  module ClassMethods
+    def prompt_start(locale:)
```

Comment:
What is the motivation for non-privatizing the refactored methods?

Reply:
They ended up being class methods so they couldn't be private anymore.

Reply:
updated this to just use a class for TranslationPrompts because there was a lot of weirdness with trying to call methods directly from a module in the rake task